resources/application-connector/values.yaml

```diff
@@ -73,9 +73,9 @@ global:
   event_service_integration_tests:
     version: "d6bbc47a"
   central_application_connectivity_validator:
-    version: "PR-11558"
+    version: "PR-11827"
   central_application_gateway:
-    version: "PR-11631"
+    version: "PR-11827"
```

Comment:
There are no changes in these components (only their charts are changed), therefore there are no new images

Reply:
I removed this change